example/full/instance.properties

```diff
@@ -1462,6 +1462,10 @@ sleeper.default.compaction.job.id.assignment.commit.async=true
 # finished jobs directly to the state store.
 sleeper.default.compaction.job.commit.async=true
 
+# This is the default for whether the asynchronous compaction commits will be sent to the batcher if
+# enabled.
```

Comment:
This would be clearer as:

"This property is the default for whether commits of compaction jobs are batched before being sent to the state store commit queue to be applied by the committer lambda. If this property is true and asynchronous commits are enabled then commits of compactions will be batched. If this property is false and asynchronous commits are enabled then commits of compactions will not be batched and will be sent directly to the committer lambda. This property can be overridden for individual tables."